inc/namespace.php

```diff
@@ -74,6 +78,80 @@ function load_metadata() {
 	} );
 }
 
+/**
+ * Add filters to use Tachyon image URLs for metadata images.
+ * Image size and crop depend on the social media type.
+ */
+function use_tachyon_img_in_metadata() {
+	add_filter( 'hm.metatags.context.default', __NAMESPACE__ . '\\metadata_img_as_tachyon' );
+	add_filter( 'hm.metatags.context.twitter', __NAMESPACE__ . '\\metadata_img_as_tachyon_twitter' );
+	add_filter( 'hm.metatags.context.opengraph', __NAMESPACE__ . '\\metadata_img_as_tachyon_opengraph' );
+}
+
+/**
+ * Update twitter metadata to use Tachyon img URL.
+ *
+ * @param array $meta Twitter metadata.
+ *
+ * @return array Twitter metadata with image using Tachyon URL, if any.
+ */
+function metadata_img_as_tachyon_twitter( array $meta ) : array {
+	return metadata_img_as_tachyon( $meta, [
+		'w'      => 1200,
+		'h'      => 600,
+		'resize' => '1200, 600', // crop
+	] );
+}
+
+/**
+ * Update opengraph metadata to use Tachyon img URL.
+ *
+ * @param array $meta opengraph metadata.
+ *
+ * @return array opengraph metadata with image using Tachyon URL, if any.
+ */
+function metadata_img_as_tachyon_opengraph( array $meta ) : array {
+	return metadata_img_as_tachyon( $meta, [
+		'w'   => 1200,
+		'h'   => 627,
+		'fit' => '1200, 627', // no crop
+	] );
+}
+
+/**
+ * Update metadata image URL to use Tachyon URL with specified image settings.
+ *
+ * @param array $meta          Metadata per social media type.
+ * @param array $img_settings Image settings: size and crop to be used in Tachyon URL.
+ *
+ * @return array Metadata with updated image URL using Tachyon, if an image is specified.
+ */
+function metadata_img_as_tachyon( array $meta, array $img_settings = [] ) : array {
+	// Stop - no image for metadata.
+	if ( ! isset( $meta['image'] ) ) {
+		return $meta;
+	}
+
+	// Default image settings.
+	$img_settings = $img_settings :? [ 'fit' => '1200, 1200' ]; // no crop
```

Comment:
```suggestion
	$img_settings = $img_settings ?: [ 'fit' => '1200,1200' ]; // no crop
```
There should be no spaces in the dimensions data, also `:?` is incorrect syntax, would've thought that would throw a warning or error.